Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -0,0 +1,169 @@
+# Detailed overview of ArangoSearch views
+
+ArangoSearch provides a fulltext search and it is only a subset of its available 
+functionality, supported via the 'text' analyzer and 'tfidf'/'bm25' [scorers](Scorers.md), 
+without impact to performance when specifying documents from different collections or 
+filtering on multiple document attributes.
+
+## View datasource
+
+The IResearch functionality is exposed to ArangoDB via the the ArangoSearch view
+API because the ArangoSearch view is merely an identity transformation applied
+onto documents stored in linked collections of the same ArangoDB database.
+In plain terms an ArangoSearch view only allows filtering and sorting of documents
+located in collections of the same database. The matching documents themselves
+are returned as-is from their corresponding collections.
+
+## Links to ArangoDB collections
+
+A concept of an ArangoDB collection 'link' is introduced to allow specifying
+which ArangoDB collections a given ArangoSearch View should query for documents
+and how these documents should be queried.
+
+An ArangoSearch Link is a uni-directional connection from an ArangoDB collection
+to an ArangoSearch view describing how data coming from the said collection should
+be made available in the given view. Each ArangoSearch Link in an ArangoSearch
+view is uniquely identified by the name of the ArangoDB collection it links to.
+An ArangoSearch view may have zero or more links, each to a distinct ArangoDB
+collection. Similarly an ArangoDB collection may be referenced via links by zero
+or more distinct ArangoSearch views. In plain terms any given ArangoSearch view
+may be linked to any given ArangoDB collection of the same database with zero or
+at most one link. However, any ArangoSearch view may be linked to multiple
+distinct ArangoDB collections and similarly any ArangoDB collection may be
+referenced by multiple ArangoSearch views.
+
+To configure an ArangoSearch view for consideration of documents from a given
+ArangoDB collection a link definition must be added to the properties of the
+said ArangoSearch view defining the link parameters as per the section
+[View definition/modification](#view-definitionmodification).
+
+## Analyzers
+
+To simplify query syntax ArangoSearch provides a concept of 
+[named analyzers](Analyzers.md) which are merely aliases for
+type+configuration of IResearch analyzers. Management of named analyzers
+is exposed via both REST, GUI and JavaScript APIs, e.g.
+
```

Comment:
`, e.g.` and then there's the end of the paragraph. Is something missing here, or can we simply delete e.g.?

Reply:
Removed.